lib/ansible/modules/system/user.py

```diff
@@ -514,7 +526,13 @@ def modify_user_usermod(self):
 
         if self.expires:
             cmd.append('--expiredate')
-            cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+            try:
+                if self.clearexpires < 0:
```

Comment:
no. conversion of parameter expires to time would triggern an exception and abort the run. line 322, branch "devel"

        if module.params['expires']:
            try:
                self.expires = time.gmtime(module.params['expires'])
            except Exception:
                e = get_exception()
                module.fail_json(msg="Invalid expires time %s: %s" %(self.expires, str(e)))